study-builder/studies/testboston/study-kits.conf

```diff
@@ -3,7 +3,22 @@
     {
       "type": "TESTBOSTON",
       "quantity": 1,
-      "needsApproval": true,
+      "needsApproval": false,
```

Comment:
Latest requirement: testboston kits no longer need approval when we launch the study.